cmd/prom-migrator/README.md

```diff
@@ -72,6 +72,9 @@ as write endpoints are somewhat constrained.
 ./prom-migrator -start=1606408552 -end=1606415752 -reader-url=<read_endpoint_url_for_remote_read_storage> -writer-url=<write_endpoint_url_for_remote_write_storage> -progress-metric-url=<read_endpoint_url_for_remote_write_storage>
 ```
 
+**Note:** Prom-migrator does not support migrating data to a HA environment. If you have an HA setup,
+try running it in non-HA mode to ingest data from prom-migrator.
```

Comment:
Instead of recommending user to run Promscale in non HA mode, let's recommend running a new Promscale connector only for the migrator and shut it down post migration. This should be ideal as it doesn't require effort's of disabling and Re-enabling HA mode.

Reply:
The problem is Prom-migrator is beyond Promscale. We know Promscale can do only for migration, but other systems might not. Hence the best is to say don't use HA while ingesting data from the migrator.